retriever.py

```diff
@@ -0,0 +1,272 @@
+import sys
+import argparse
+import datetime
+import io
+import os
+import xml
+import requests
+from fs import open_fs
+import PIL.Image
+
+import mslib
+import mslib.utils
+import mslib.msui
+import mslib.msui.mpl_map
+import mslib.utils.qt
+import mslib.utils.thermolib
+from mslib.utils.config import config_loader, read_config_file
+from mslib.utils.units import units
+from mslib.msui import wms_control
+import matplotlib.pyplot as plt
+
+
+TEXT_CONFIG = {
+    "bbox": dict(boxstyle="round", facecolor="white", alpha=0.5, edgecolor="none"),
+    "fontweight": "bold", "zorder": 4, "fontsize": 6, "clip_on": True}
+
+
+def load_from_ftml(filename):
```

Comment:
Ripped from
mslib/msui/flighttrack.py:608ff

The whole FTML reading should be refactored to use the "general" fileio plugin structure. That would be a cleaner design than the current (historical) separation.

See mslib/plugins/io